releases/client/2.0/settings

```diff
@@ -0,0 +1,3 @@
+FULLVERSION="2.0-rc1"
```

Comment:
Does the client care about the exact version or can we simply say `FULLVERSION=$FOREMAN_VERSION`?

Reply:
no, it does not care, you're right.